spec/menu_spec.rb

```diff
@@ -0,0 +1,15 @@
+require './docs/menu.rb' 
+
+describe 'Menu' do
+    it 'should return a hash with 4 item inside' do  
+        menu = Menu.new
```

Comment:
You could use let(:menu) { described_class.new } at the top of your describe block which would mean that you wouldnt need to call it multiple times.

Reply:
It might be worth changing this test so that it is flexible with any menu. If another item is added or a price is changed, this test would fail.